octopoes/tests/robot/CeleryMonitor.py

```diff
@@ -11,13 +11,17 @@
 def get_app(queue_uri: str) -> Celery:
     config = {
         "broker_url": queue_uri,
+        "broker_pool_limit": 0,
+        "worker_concurrency": 1,
+        "worker_prefetch_multiplier": 0,
         "result_backend": f"rpc://{queue_uri}",
         "task_serializer": "json",
         "result_serializer": "json",
         "event_serializer": "json",
         "accept_content": ["application/json", "application/x-python-serialize"],
         "result_accept_content": ["application/json", "application/x-python-serialize"],
         "task_queues": ("octopoes",),
+        "task_alway_eager": True,
```

Comment:
This is the config for the celerymonitor / the part running inside the robot tests, so won't do anything as far as I understand it. The tasks are created and executed inside octopoes api and api worker and not the robot tests.

I also think it is better if we don't configure celery in the integration tests differently from how it is normally run unless it is really necessary.

Reply:
Addressed in 5dfb5afad4312791bc4b3dc6c6409884247f1425.